server/index.js

```diff
@@ -90,6 +90,21 @@ app.get("/v2/query", async (req, res) => {
   res.send({ location, results });
 });
 
+app.get("/v2/queryByStoreId", async (req, res) => {
+  const { query } = req;
+  let params = {
+    storeId: query.storeId,
+    radius: query.radius,
+    page: query.page
+  }
+  let results = await stores.findNearbyStoreByStoreId(params)
```

Comment:
it seems that `results` can be an empty array. can we check for `results.length` before accessing `results[0]`